src/app.js

```diff
@@ -22,9 +22,8 @@ app.use(bodyParser.json({ type: 'application/vnd.api+json' }));
 app.use(methodOverride('X-HTTP-Method-Override'));
```

Comment:
all of app.use are also middleware(just not custom), so they must be applied within the method from index.js too